src/sds_data_model/constants.py

```diff
@@ -1,7 +1,7 @@
 from itertools import product
 from typing import Tuple
 
-BNG = "OSGB36 / British National Grid"
+BNG = "OSGB 1936 / British National Grid"
```

Comment:
```suggestion
BNG = "OSGB36 / British National Grid"
```
Rollback change to resolve conflict. [Solution](https://github.com/Defra-Data-Science-Centre-of-Excellence/sds-data-model/blob/564956f3888d50e56e46606a97bf3b97d890773c/src/sds_data_model/constants.py#L5) already merged in #20.